### DISCUSSION TITLE
طراحی زیر ساخت جمع آوری کمک مالی برای حمایت اعتصاب کنندگان

### DISCUSSION THREAD

**Discussion by moh-sanati** [WeAreMahsaAmini/Core#470] [Ideas]
سلام
با توجه به اهمیت بالای اعتصابات و استمرار آن ، از دوستان متخصص دعوت می کنم درباره طراحی زیر ساختی برای جمع آوری کمک مالی از تمام افراد در سرتاسر دنیا و مدیریت و پرداخت ماهانه به اعتصاب کنندگان هم فکری و کمک کنند.
چند مورد از مواد مهم مینویسم دوستان اضافه کنند:
1)امکان ناشناس بودن افراد کمک کننده
2)امکان دریافت کمک های مالی از داخل و خارج از کشور
3)زیر ساخت غیر قابل شناسایی و فیلتر
4)شناسایی و تایید افراد اعتصاب کننده که نیاز به کمک مالی دارند

**Comment by FreedomFighterOfIran** [comment]
سلام
من تکنولوژی ای برای این کار میشناسم که یه دیسکاشن جدید باز کردم برای معرفیش
https://github.com/WeAreMahsaAmini/Core/discussions/482

**Reply by moh-sanati** [reply]
شما چقدر در حوزه بلاکچین و زیر ساخت آن کار کردید؟بیشتر از نظر فنی و پیاده سازی

**Reply by FreedomFighterOfIran** [reply]
> شما چقدر در حوزه بلاکچین و زیر ساخت آن کار کردید؟بیشتر از نظر فنی و پیاده سازی

مدت زیادی هست که در این زمینه فعالم. ادامه ی صحبتی اگر هست کاش تو همون دیسکاشنی که فرستادم بگید 🙂
